packages/calling-component-bindings/review/beta/calling-component-bindings.api.md

```diff
@@ -197,7 +207,7 @@ export const useCallClient: () => StatefulCallClient;
 export const useCallingHandlers: <PropsT>(component: (props: PropsT) => ReactElement | null) => Common<CallingHandlers, PropsT> | undefined;
 
 // @public
-export const useCallingPropsFor: <Component extends (props: any) => JSX.Element>(component: Component) => GetCallingSelector<Component> extends (props: any) => any ? ReturnType<GetCallingSelector<Component>> & Common<CallingHandlers, Parameters<Component>[0]> : undefined;
+export const useCallingPropsFor: <Component extends (props: any) => JSX.Element>(component: Component) => FilterUndefined<(AreEqual<Component, (props: VideoGalleryProps) => JSX.Element> extends true ? VideoGallerySelector : undefined) | (AreEqual<Component, (props: DevicesButtonProps) => JSX.Element> extends true ? DevicesButtonSelector : undefined) | (AreEqual<Component, (props: MicrophoneButtonProps) => JSX.Element> extends true ? MicrophoneButtonSelector : undefined) | (AreEqual<Component, (props: CameraButtonProps) => JSX.Element> extends true ? CameraButtonSelector : undefined) | (AreEqual<Component, (props: ScreenShareButtonProps) => JSX.Element> extends true ? ScreenShareButtonSelector : undefined) | (AreEqual<Component, (props: ParticipantListProps) => JSX.Element> extends true ? ParticipantListSelector : undefined) | (AreEqual<Component, (props: ParticipantsButtonProps) => JSX.Element> extends true ? ParticipantsButtonSelector : undefined) | (AreEqual<Component, (props: EndCallButtonProps) => JSX.Element> extends true ? EmptySelector : undefined) | (AreEqual<Component, (props: ErrorBarProps) => JSX.Element> extends true ? ErrorBarSelector : undefined)> extends (props: any) => any ? ReturnType<FilterUndefined<(AreEqual<Component, (props: VideoGalleryProps) => JSX.Element> extends true ? VideoGallerySelector : undefined) | (AreEqual<Component, (props: DevicesButtonProps) => JSX.Element> extends true ? DevicesButtonSelector : undefined) | (AreEqual<Component, (props: MicrophoneButtonProps) => JSX.Element> extends true ? MicrophoneButtonSelector : undefined) | (AreEqual<Component, (props: CameraButtonProps) => JSX.Element> extends true ? CameraButtonSelector : undefined) | (AreEqual<Component, (props: ScreenShareButtonProps) => JSX.Element> extends true ? ScreenShareButtonSelector : undefined) | (AreEqual<Component, (props: ParticipantListProps) => JSX.Element> extends true ? ParticipantListSelector : undefined) | (AreEqual<Component, (props: ParticipantsButtonProps) => JSX.Element> extends true ? ParticipantsButtonSelector : undefined) | (AreEqual<Component, (props: EndCallButtonProps) => JSX.Element> extends true ? EmptySelector : undefined) | (AreEqual<Component, (props: ErrorBarProps) => JSX.Element> extends true ? ErrorBarSelector : undefined)>> & Common<CallingHandlers, Parameters<Component>[0]> : undefined;
```

Comment:
This seems to be a quirk of api-extractor that inlines type definitions that are simple just aliases, but not those that contain more complex logic.

So in this case, because `GetSelector` is defined as `FilterUndefined<....>`, api-extractor inlines the definition in several places.